pkg/hydrate/controller.go

```diff
@@ -75,23 +75,23 @@ func (h *Hydrator) Run(ctx context.Context) {
 	defer runTimer.Stop()
 	rehydrateTimer := time.NewTimer(h.RehydratePeriod)
 	defer rehydrateTimer.Stop()
-	absSourceDir := h.absSourceDir()
+	sourcePath := h.sourcePath()
 	var hydrateErr HydrationError
 	var srcCommit string
-	var syncDir cmpath.Absolute
+	var syncPath cmpath.Absolute
```

Comment:
Could you document the difference between `sourcePath` and `syncPath`?

Reply:
done